src/conn/conn_open.c

```diff
@@ -275,5 +277,8 @@ __wti_connection_workers(WT_SESSION_IMPL *session, const char *cfg[])
     /* Start the checkpoint cleanup thread. */
     WT_RET(__wt_checkpoint_cleanup_create(session, cfg));
 
+    __wt_verbose_info(
+      session, WT_VERB_RECOVERY, "%s", "WiredTiger utility threads are started successfully");
```

Comment:
Can you please rephrase it to "WiredTiger utility threads started successfully." ?

Reply:
Done.